content/en/real_user_monitoring/session_replay/troubleshooting.md

```diff
@@ -19,11 +19,7 @@ If you experience unexpected behavior with Datadog Session Replay, use this page
 
 ### Some HTML elements are not visible at replay
 
-Session Replay does not support the following HTML elements: `iframe`, `video`, `audio`, or `canvas`. Session Replay does not support Web Components nor Shadow DOM.
-
-{{< callout url="#" btn_hidden="true" >}}
-Shadow DOM support is in beta. To request access, contact Datadog support at support@datadoghq.com.
-{{< /callout >}}
+Session Replay does not support the following HTML elements: `iframe`, `video`, `audio`, or `canvas`. 
```

Comment:
in this case it's just shadow DOM so it would be a bit odd to just list the one we support. i realize we don't usually list what's not supported but i think in this case it's more relevant

Reply:
@jamiemilstein That makes sense to me, though I just wanted to make sure we're talking about the same thing here - is Shadow DOM a HTML element?

Reply:
it could be yep! it's likely not a single element but instead a group of them. i think fullstory's blog on it does a nice job giving an overview of the tech https://www.fullstory.com/blog/shadow-dom/